var/spack/repos/builtin/packages/librsvg/package.py

```diff
@@ -12,21 +12,22 @@ class Librsvg(AutotoolsPackage):
     homepage = "https://wiki.gnome.org/Projects/LibRsvg"
     url      = "https://download.gnome.org/sources/librsvg/2.44/librsvg-2.44.14.tar.xz"
 
-    version('2.44.14', sha256='6a85a7868639cdd4aa064245cc8e9d864dad8b8e9a4a8031bb09a4796bc4e303')
```

Comment:
Can we keep the old version?

Reply:
The old version didn't compile for me.  Does it still work for you?

Reply:
I didn't try to build it, but usually we don't remove old versions of libraries when new ones are available. We should start doing that at some point, but policies are still under discussion #15896